package-lock.json

```diff
@@ -1,8 +1,3037 @@
 {
   "name": "trivy-vulnerability-scanner",
-  "version": "0.2.8",
-  "lockfileVersion": 1,
+  "version": "0.7.4",
+  "lockfileVersion": 2,
```

Comment:
how is the version determined? In the changelog I see it's at 0.8.0. I'm fine either way as long as they are in sync.

Reply:
I have no idea what happened there. I think npm bumped it automagically for me. I've gone with 0.3.0. I think its probably a major change but its not breaking, and we don't seem to like getting to v1 too quickly :-)